sbt-plugin-test/build.sbt

```diff
@@ -182,10 +182,30 @@ lazy val jsDependenciesTest = project.settings(versionSettings: _*).
           "compressed/history.js appears before foo.js")
 
       streams.value.log.info("jsDependencies resolution test passed")
+
+      // Regression test for issue #2243
+      val pack = (packageJSDependencies in Compile).value
+      val minPack = (packageMinifiedJSDependencies in Compile).value
+      assert((artifactPath in (Compile, packageJSDependencies)).value.exists)
+      assert((artifactPath in (Compile, packageMinifiedJSDependencies)).value.exists)
+      streams.value.log.info("Regression test for issue #2243 passed")
     })
   )): _*).
   dependsOn(jetty9) // depends on jQuery
 
+lazy val jsNoDependenciesTest = project.settings(versionSettings: _*).
+    enablePlugins(ScalaJSPlugin).
+    settings(inConfig(Compile)(Seq(
+      packageJSDependencies <<= packageJSDependencies.dependsOn(Def.task {
+        // Regression test for issue #2243
+        val pack = (packageJSDependencies in Compile).value
+        val minPack = (packageMinifiedJSDependencies in Compile).value
+        assert((artifactPath in (Compile, packageJSDependencies)).value.exists)
+        assert((artifactPath in (Compile, packageMinifiedJSDependencies)).value.exists)
+        streams.value.log.info("Regression test for issue #2243 passed")
```

Comment:
Added two sets of regression tests, one for the case where there are dependencies and one for the case where there are none.